### PR TITLE
aruha-494: fixed errors and warnings according to swagger online editor

### PR DESCRIPTION
**Pull request by adyach**

### CODE REVIEW

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -112,8 +112,6 @@ paths:
     get:
       tags:
         - schema-registry-api
-      security:
-        - oauth2: ['nakadi.event_type.read']
```
Comment (lmontrieux) [inline api/nakadi-event-bus-api.yaml]: Why is this removed?

File: api/nakadi-event-bus-api.yaml [hunk 2]
```diff
@@ -200,8 +198,6 @@ paths:
...
```
Comment (lmontrieux) [inline api/nakadi-event-bus-api.yaml]: Why is this removed?

File: api/nakadi-event-bus-api.yaml (hunk 1 above)
Comment (adyach) [inline api/nakadi-event-bus-api.yaml]: It was decided if user has oauth token he can read.

File: api/nakadi-event-bus-api.yaml (hunk 2 above)
Comment (adyach) [inline api/nakadi-event-bus-api.yaml]: It was decided if user has oauth token he can read.

File: api/nakadi-event-bus-api.yaml (hunk 1 above)
Comment (adyach) [inline api/nakadi-event-bus-api.yaml]: It was wrongly added by me.

**Comment by adyach** [comment]
@lmontrieux it is used in /subscriptions/{subscription_id}/evens and response description is wrong there. I found it out comparing the code and definition of SubscriptionEventStreamBatch. I will fix it.

**Comment by lmontrieux** [comment]
:shipit:

**Comment by adyach** [comment]
@antban @v-stepanov @rcillo pls, review/approve the PR, I can not merge it because @lmontrieux is not approver in .zappr. 
Created https://github.com/zalando/nakadi/pull/507 to fix this.

**Comment by v-stepanov** [comment]
:shipit:
